python/lsst/ts/gis/csc.py

```diff
@@ -2,18 +2,17 @@
 
 import asyncio
 
+from importlib_metadata import version
 from lsst.ts import salobj, utils
-from pymodbus.datastore import (
-    ModbusSequentialDataBlock,
-    ModbusServerContext,
-    ModbusSlaveContext,
-)
-from pymodbus.server import ModbusTcpServer
+from packaging.version import parse
+from pymodbus.server.simulator.http_server import ModbusSimulatorServer
 
 from . import __version__
 from .component import GISComponent
 from .config import CONFIG_SCHEMA
 
+FILE = "python/lsst/ts/gis/data/setup.json"
```

Comment:
There are many examples in our python code that show how to use pathlib to get to a file at runtime. Hardcoded paths like this are not safe. You should probably do something like this instead:
```
FILE = pathlib.Path(__file__).resolve().parents[0] / "data" / "setup.json"
```